src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -867,234 +1099,8 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
-			),
-			'BP_MEMBERS_SLUG' => array(
-				'label' => 'BP_MEMBERS_SLUG (deprecated)',
-				'value' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : 'undefined',
-			),
-			'BP_GROUPS_SLUG' => array(
-				'label' => 'BP_GROUPS_SLUG (deprecated)',
-				'value' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : 'undefined',
-			),
-			'BP_MESSAGES_SLUG' => array(
-				'label' => 'BP_MESSAGES_SLUG (deprecated)',
-				'value' => defined( 'BP_MESSAGES_SLUG' ) ? BP_MESSAGES_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_MESSAGES_SLUG' ) ? BP_MESSAGES_SLUG : 'undefined',
-			),
-			'BP_NOTIFICATIONS_SLUG' => array(
-				'label' => 'BP_NOTIFICATIONS_SLUG (deprecated)',
-				'value' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : 'undefined',
-			),
-			'BP_BLOGS_SLUG' => array(
-				'label' => 'BP_BLOGS_SLUG (deprecated)',
-				'value' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : 'undefined',
-			),
-			'BP_FRIENDS_SLUG' => array(
-				'label' => 'BP_FRIENDS_SLUG (deprecated)',
-				'value' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : 'undefined',
-			),
-			'BP_ACTIVITY_SLUG' => array(
-				'label' => 'BP_ACTIVITY_SLUG (deprecated)',
-				'value' => defined( 'BP_ACTIVITY_SLUG' ) ? BP_ACTIVITY_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_ACTIVITY_SLUG' ) ? BP_ACTIVITY_SLUG : 'undefined',
-			),
-			'BP_SETTINGS_SLUG' => array(
-				'label' => 'BP_SETTINGS_SLUG (deprecated)',
-				'value' => defined( 'BP_SETTINGS_SLUG' ) ? BP_SETTINGS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_SETTINGS_SLUG' ) ? BP_SETTINGS_SLUG : 'undefined',
-			),
-			'BP_XPROFILE_SLUG' => array(
-				'label' => 'BP_XPROFILE_SLUG (deprecated)',
-				'value' => defined( 'BP_XPROFILE_SLUG' ) ? BP_XPROFILE_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_XPROFILE_SLUG' ) ? BP_XPROFILE_SLUG : 'undefined',
-			),
-			'BP_FORUMS_SLUG' => array(
-				'label' => 'BP_FORUMS_SLUG (deprecated)',
-				'value' => defined( 'BP_FORUMS_SLUG' ) ? BP_FORUMS_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_FORUMS_SLUG' ) ? BP_FORUMS_SLUG : 'undefined',
-			),
-			'BP_SEARCH_SLUG' => array(
-				'label' => 'BP_SEARCH_SLUG (deprecated)',
-				'value' => defined( 'BP_SEARCH_SLUG' ) ? BP_SEARCH_SLUG : __( 'Undefined', 'buddypress' ),
-				'debug' => defined( 'BP_SEARCH_SLUG' ) ? BP_SEARCH_SLUG : 'undefined',
-			),
+			$bp_settings,
```

Comment:
Unrelated to this change, but I noticed some values are really weird when copying the values.

![CleanShot 2024-03-02 at 10 04 15@2x](https://github.com/buddypress/buddypress/assets/19148962/f405e841-90f2-40a5-8259-c486c2dee813)

Reply:
I don't remember very well, but I might have added these exclamation marks because of the way these options are formulated / handled inside BuddyPress. Without the exclamation mark for example:  `bp-disable-avatar-uploads: Yes` would mean avatar uploads are disabled but I don't think it's the case. I'll look at i more closely.

Reply:
Do we save the "Yes" string in the database? If not, we should add the actual value for the copied version.